admin/src/app/about/about.component.ts

```diff
@@ -0,0 +1,13 @@
+import { Component } from '@angular/core';
+
+@Component({
+  standalone: true,
+  selector: 'app-about',
+  templateUrl: './about.component.html',
+  styleUrls: ['./about.component.scss']
+})
+export class AboutComponent {
```

Comment:
Hi Ian, just a small question, wondering if this about component is just about the training material, do we still want to call it about or call it training? wondering if later we'll add more information to this page, or just for training.

Reply:
Hi Catherine,  yes, I thought that a little before.
Public site has "**About**" but Admin site did not have. I reuse it from the public site since the design layout were to be consistent with public "**About**" site.
To my view, "About" component is just a general container that might have different sections within it (like at public site) for users' references, and being consistent on both admin and public for components' name may make developers easy to related. 
And the UI name is just a alias to the component's router, so it can be changed easily. **Unless** we like to have (in the future) one "**About**" and one "**Training**" separated on the header tabs.
The tab name Olga design calls it "**Training Modules**", if all prefers, I can refactor the component (not the UI name) to "**Training**" but I am not sure if Julius would like to keep it consistent later to call it "**About**" . Either naming is ok for me. @OlgaLiber2

Reply:
I'm fine with it, but maybe we can add a comment in the about component that for now this is just used for training module?

Reply:
Hi @MCatherine1994 , comment added to the component. :)
And holder's training link updated.